modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx

```diff
@@ -70,7 +71,21 @@ export default function VisualizationModes(props: IFDSViewSectionProps) {
 									className="px-0"
 									key={visualizationMode.visualizationModeId}
 								>
-									<Component {...props} />
+									<ClayLayout.ContentCol className="c-gap-4 cards-visualization-mode">
+										<ClayAlert
+											displayType="info"
+											title={`${Liferay.Language.get(
+												'info'
+											)}:`}
+											variant="stripe"
+										>
+											{Liferay.Language.get(
+												'this-visualization-mode-will-not-be-shown-until-you-assign-at-least-one-field-to-a-card-element'
```

Comment:
This is the Cards info message. It does not make sense for all different visualization modes.